lib/querystring.js

```diff
@@ -90,8 +90,11 @@ for (var i = 0; i < 256; ++i)
 QueryString.escape = function(str) {
   // replaces encodeURIComponent
   // http://www.ecma-international.org/ecma-262/5.1/#sec-15.1.3.4
-  if (typeof str !== 'string')
-    str += '';
+  if (typeof str !== 'string') {
+    if (typeof str === 'object')
+      str = String(str);
+    else str += '';
```

Comment:
No, I meant this as a style nit: moving the `str += '';` to the next line instead of on the same as line as the `else`.